tapdb/multiverse.go

```diff
@@ -960,7 +960,9 @@ func (b *MultiverseStore) FetchProof(ctx context.Context,
 		return proofs[0].Leaf.RawProof, nil
 	}
 
-	file, err := proof.FetchProofProvenance(ctx, originLocator, fetchProof)
+	file, err := proof.FetchProofProvenance(
+		ctx, nil, originLocator, fetchProof,
```

Comment:
IIUC here the local archive is `nil` as we're already in the DB?

Reply:
Correct. Makes the code a bit harder to read but allows us to re-use the same algorithm for fetching proof individually from a universe in multiple places.